source/guides/clear/mixer.rst

```diff
@@ -280,6 +287,29 @@ inherit upstream bundles.
       mixer bundle add editors
       mixer bundle edit editors
 
+#. We can build a mix from RPM packages (a.k.a ``package bundles``) 
+   as well as from ``content bundles``. For building a mix from RPM
+   packages, specify the package name in the bundle definition file, as
+   shown below. 
+
+   .. code-block:: bash
+
+      TBD
+
+#. For building a mix from ``content bundles``, specify the file 
+   path to the content in the bundle definition file, using the keyword 
+   :gui:`content()`. See the example below. 
```

Comment:
I think you mean :guilabel: not :gui: (line 301).

Reply:
Great catch. Thank you.